.github/workflows/smart-version-check.yml

```
@@ -0,0 +1,142 @@
+        
+    - name: 🚨 Check for version conflicts
+      if: github.event_name == 'pull_request'
+      env:
+        GEMINI_API_KEY: ${{ secrets.GEMINI_API_KEY }}
+      run: |
+        echo "🔧 Checking for version conflicts in PR..."
+        
+        # Get base branch version
+        git checkout ${{ github.base_ref }}
+        BASE_VERSION=$(python -c "from aider_lint_fixer import __version__; print(__version__)" 2>/dev/null || echo "unknown")
+        
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)